CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 ## 3.2.1(unreleased)
```

Comment:
the new version should be 3.3.0 since its a new resource added to it as per semantic versioning

Reply:
@soodpr Corrected